php-cs-fixer.ta.php

```diff
@@ -1,13 +1,22 @@
 <?php
 
+use PhpCsFixer\RuleSet\Sets\PhpCsFixerSet;
+
+$csFixerSet = new PhpCsFixerSet();
+
 return (new PhpCsFixer\Config('standards'))
     ->setRiskyAllowed(true)
     ->setRules([
         '@PhpCsFixer' => true,
         '@Symfony:risky' => true,
-        '@PHP80Migration' => true,
-        '@PHP80Migration:risky' => true,
-        '@PHPUnit84Migration:risky' => true,
+        'blank_line_before_statement' => [
+            // don't put lines between "case" statements
+            'statements' => array_filter(
+                $csFixerSet->getRules()['blank_line_before_statement']['statements'],
+                static fn (string $statement): bool => $statement !== 'case',
+            )
+        ],
+        'blank_line_between_import_groups' => false,
```

Comment:
Prevents these messy imports:

```php
class Foo
{
    use const AAAA;
    use const BBB;

    use Acme;

    use function DDD;

    use AAC;

    use function CCC\AA;

    use Bar;
}
```
(literally an example of the configurator, I don't understand why they think this is a nice configuration)